www/src/components/Menu.tsx

```diff
@@ -185,43 +201,61 @@ function Menu() {
         <Collapsible
           styling="basic"
           title="Guides"
+          defaultOpen={pathname.startsWith('/guides')}
         >
           <ul className="list-unstyled">
             <li>
-              <Link to="/guides/installation-and-usage">
+              <Link
+                className={classNames({ active: pathname.endsWith('getting-started') })}
+                to="/guides/getting-started"
+              >
                 Getting started
               </Link>
             </li>
             <li>
-              <a
+              <Hyperlink
+                destination="https://openedx.atlassian.net/wiki/spaces/BPL/pages/1773502564/Component+Contribution+Process"
                 target="_blank"
-                rel="noopener noreferrer"
-                href="https://openedx.atlassian.net/wiki/spaces/BPL/pages/1773502564/Component+Contribution+Process"
               >
                 Contributing
-              </a>
+              </Hyperlink>
             </li>
           </ul>
         </Collapsible>
         <Collapsible
           styling="basic"
           title="Foundations"
+          defaultOpen={pathname.startsWith('/foundations')}
         >
           <ul className="list-unstyled foundations-list">
-            {foundationLinks.map(link => (
-              <li key={link}>
-                <Link to={`/foundations/${link.toLowerCase().trim()}`}>{link.replace(/-/g, ' ')}</Link>
-              </li>
-            ))}
+            {foundationLinks.map(link => {
+              const isActive = pathname.endsWith(link.toLowerCase()) && !pathname.endsWith(`brand-${link.toLowerCase()}`);
+              return (
+                <li key={link}>
+                  <Link
+                    className={classNames({ active: isActive })}
+                    to={`/foundations/${link.toLowerCase().trim()}`}
+                  >
+                    {link.replace(/-/g, ' ')}
+                  </Link>
+                </li>
+              );
+            })}
```

Comment:
Did you add ``!pathname.endsWith(`brand-${link.toLowerCase()}`)`` to differentiate between icons and brand-icons pages? If so, I think you can avoid doing that with single check ``className={classNames({ active: pathname.endsWith(`/${link.toLowerCase()}`) })}``


Also, can you please refactor `foundationLinks` and make it an array of objects instead of strings? e.g.
```js
const foundationLinks = [
  {
    label: 'Colors'.
    link: '/foundations/colors/',
  }
  ...
]
```

this would make this code a lot clearer, right now the logic is very confusing with `.toLowerCase()`, `.trim()`, `.replace(/-/g, ' ')` added everywhere.

Reply:
Yes, you're right, it will look much cleaner this way.